### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by rpgayatri**
Refactoring of create-payments-company 
Refactoring of payments-company update
Refactoring of bulk user and company update

### CODE REVIEW

File: src/main/resources/rules/rulesCurrent/prj_channel40/TO_UPDATE_WORKFLOWS/Payments/AssemblyAttributeChange/CompanyAttributeChange/10_CheckIfUserCompanyExistsForUpdation.drl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    	rules: QRules(!isState("LOOP_COMPANY_UPDATE") )
+    then
+		rules.header();
+		rules.setState("LOOP_COMPANY_UPDATE");
+		
+		BaseEntity userBe = rules.getUser();
+		BaseEntity companyBe = rules.getParent(userBe.getCode(), "LNK_STAFF"); 
+		
+		if(companyBe != null) {
+			drools.setFocus("payments-company-update");
```
Comment (DevJS) [inline src/main/resources/rules/rulesCurrent/prj_channel40/TO_UPDATE_WORKFLOWS/Payments/AssemblyAttributeChange/CompanyAttributeChange/10_CheckIfUserCompanyExistsForUpdation.drl:22]: really good

File: src/main/resources/rules/rulesCurrent/prj_channel40/TO_UPDATE_WORKFLOWS/Payments/PaymentsCompanyCreationWorkflow/10_CheckIfUserAlreadyHasAssemblyCompany.drl [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package life.genny.rules;
+
+import life.genny.rules.QRules;
+
+
+rule "Payments Company Creation Step 1 : Does user already have assembly company attribute"
+	agenda-group "payments"
+	no-loop true
+	when 
+		rules: QRules(isState("IS_ASSEMBLY_USER_CREATED") && !isState("IS_ASSEMBLY_COMPANY_CREATED") && !isState("LOOP_CHECK_ASSEMBLY_USER_COMPANY_CREATED"))
```
Comment (DevJS) [inline src/main/resources/rules/rulesCurrent/prj_channel40/TO_UPDATE_WORKFLOWS/Payments/PaymentsCompanyCreationWorkflow/10_CheckIfUserAlreadyHasAssemblyCompany.drl:10]: do we want to put assembly in the states? :)

Comment (rpgayatri) [reply src/main/resources/rules/rulesCurrent/prj_channel40/TO_UPDATE_WORKFLOWS/Payments/PaymentsCompanyCreationWorkflow/10_CheckIfUserAlreadyHasAssemblyCompany.drl:10]: Yes, for now. I am refactoring all the payment method calls and handling errors. The states will change and rules will get smaller for the workflow refactor.

**Review by DevJS — APPROVED** [review]
ok

**Pull request merged**